server/catgenome/src/test/java/com/epam/catgenome/manager/aws/S3ManagerTest.java

```diff
@@ -31,22 +31,26 @@
 import java.util.Date;
 
 import com.amazonaws.services.s3.AmazonS3;
+import com.epam.catgenome.common.AbstractManagerTest;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mockito;
+import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
+
 @RunWith(SpringJUnit4ClassRunner.class)
-public class S3ManagerTest {
+@ContextConfiguration({"classpath:applicationContext-test.xml"})
+public class S3ManagerTest extends AbstractManagerTest {
```

Comment:
This test doesn't have to work with context. Since you don't autowire anything here, it can be a regular test class